docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
```

Comment:
Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
the internet, and aren't centrally controlled by anyone. Unlike using a bank, this means nobody
can seize, block or take control of your money. Instead, Bitcoin is run and verified collectively by
its users, as a peer-to-peer network. Let's get started and explain the history behind Bitcoin
and how it works.

 ## Background and transactions

 Bitcoin was first described in the Bitcoin whitepaper, by Satoshi Nakamoto, its first developer.
Before Bitcoin, digital currencies and payment systems all relied on trust, by either trusting a
company, or an individual. This made it easy for these trusted persons to block people's money,
or succumb to Government intervention. Satoshi figured out a way to overcome that, by inventing
a data structure called the blockchain, which is distributed between all users in a peer-to-peer network.

 Users of the Bitcoin network create transactions, and then digitally sign them with cryptographic
signatures. A transaction contains all the information necessary to send the bitcoins to its recipient.
Once the transaction is ready, it gets broadcasted to the Bitcoin peer-to-peer network so that it can
be included into the blockchain by the Bitcoin miners, a group of users and businesses who use special
hardware to solve really complex puzzles to secure transactions and extend the blockchain. But what
really is the blockchain?

 ## Blockchain

 The blockchain is a chain of blocks, created by the Bitcoin miners by solving computationally
difficult puzzles using their specialised hardware. Each block is generated approximately every
ten minutes, and contains transactions between users. Miners compete to solve blocks so they can
claim the block reward and transaction fees. The block reward is a payout given to the miner that
solves the block, the payout amount halves every four years and is the way Bitcoin bootstraps its
circulating supply.

 Miners listen for incoming transactions between users, place them into blocks, and then compete
with one another to solve the computationally challenging puzzle. The first miner to solve the
puzzle announces their block to the network, and that block gets added to the distributed blockchain.
Once the block is added, other miners then abandon the old block they were working on, and create a
new block which builds on top of the recently announced block. This process builds a chain of blocks,
the blockchain. Every two weeks, the network uses an algorithm to limit the rate of block creation
to an average of one block every ten minutes.

 ## Fees and the role of miners

 The amount of space available for transactions in each block is limited, so users pay transaction
fees to bid for space in the next block for their transaction. This means Bitcoin's transaction fees
can be higher during periods of heavy use, and lower during quieter times. The transaction fees are
always changing, but most Bitcoin wallet software uses smart algorithms to figure out the correct fee
to pay to get your transaction processed by the miners. It costs nothing to receive Bitcoin, the
transaction fee is only paid by the sender.

 The transaction fee goes to the miner who mined the block the transaction was included in. It's
important to understand that even though miners collect a fee, and create blocks, they do not
control the Bitcoin network. The blocks miners create are verified by the users who run the
Bitcoin software. Amongst other things, the software checks any new blocks to make sure the miner
isn't cheating by claiming a higher payout then they're entitled too or reversing any transactions.
Therefore miners are unlikely to attempt to misbehave. Miners just use their specialised hardware to
work on behalf of the users and must follow the rules of the network.

 ## Ending Summary

 Bitcoin makes possible many new use cases that weren't possible before. Online shops can use
Bitcoin to receive payment for their goods, without worrying about charge backs and fraud.
Unlike banks and credit cards, you don't need to hand over any personal information to begin
using Bitcoin. The Bitcoin network never sleeps, and functions worldwide, with little to no downtime.
Through the power of Bitcoin, a business man in Russia can send bitcoins to a programmer in America,
without going through a middle man or asking anyone for permission.

 People living in countries suffering from hyperinflation can use Bitcoin as a store of value,
boosting their economic freedom. Bitcoin also has features which make possible a limited form
of smart contracting. Bitcoin is revolutionising finance the same way the web changed publishing.
You can get involved today, by downloading your first wallet. Visit Bitcoin.org for more information
to learn more about Bitcoin.